programming-in-scala/src/test/scala/org/programingscala/chapter06/RationalSpec.scala

```diff
@@ -0,0 +1,18 @@
+package org.programingscala.chapter06
+
+import org.scalatest.flatspec.AnyFlatSpec
+
+//people understand how to use your code by looking at your test
+
+class RationalSpec extends AnyFlatSpec {
+
+    "Rational" should "add two positive Rational numbers" in{
+        val oneHalf = new Rational(1, 2)
+        val r = new Rational(1, 2)
+
+        val sum = oneHalf + r
+
+        assert(sum === new Rational(2, 2))
+    }
+    
```

Comment:
where are all the other test cases?  we spend a lot time to go through this 1 test case, I expect you to add all other test cases.

Reply:
> where are all the other test cases? we spend a lot time to go through this 1 test case, I expect you to add all other test cases.

i was working yesterday ill add all the test case today